src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -492,6 +493,14 @@ public function checkAnnotations(\ReflectionClass $refl, string $class): array
                         self::$method[$class] = self::$method[$use];
                     }
                 } elseif (!$refl->isInterface()) {
+                    if (!strncmp($vendor, str_replace('_', '\\', $use), $vendorLen)
+                        && 0 === strpos($className, 'Symfony\\')
+                        && class_exists(InstalledVersions::class)
```

Comment:
```suggestion
                        && (!class_exists(InstalledVersions::class) || 'symfony/symfony' !== InstalledVersions::getRootPackage()['name'])
```

Let's not annoy Composer 1 users either. 🙂 

Alternatively, we could check if `Symfony\Bundle\FullStack` is defined.

Reply:
good idea, updated!